src/main/resources/templates/placeOrder.html

```diff
@@ -17,121 +17,121 @@
             <h1 class="display-4">Place an Order</h1>
         </div>
     </div>
-</main>
 
-<div class="container">
-    <form>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="inlineFormInputName">Name</label>
-                <input type="email" class="form-control" id="inlineFormInputName" placeholder="Company or name">
+    <div class="container outerContentWrapper">
```

Comment:
Added the outerContentWrapper CSS class to constrain the content and keep it from overflowing into other elements, such as the footer.